README.md

```diff
@@ -22,8 +22,20 @@ Plugin for viewing streams located on the ant-media server. There is also a func
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 ## Installation
 
-```sh
+```bash
 npm install --save @antmedia/videojs-webrtc-plugin
+
+OR
+
+npm install
```

Comment:
This command and below running or testing locally of this project. 

To proceed
- Put it into a new header like Quick Start
npm install 
npm run build
npm run start 

and also mention which URL to change to see that  it's working. I mean there will be full instructions that zero beginner can do it easily.